src/git-tabs.js

```diff
@@ -1,55 +1,17 @@
-var CompositeDisposable = require('atom').CompositeDisposable;
+const git = require('./git');
 
-var git = require('./git');
-
-var GitTabs = {
-
-    subscriptions: null,
```

Comment:
as far as i know we're still going to need `subscriptions`.
this is how we're going to interface with a user adding another tab/closing out of the editor etc